utils/chemiscope_input.py

```diff
@@ -210,3 +210,54 @@ def write_chemiscope_input(filename, frames, meta=None, extra=None, cutoff=None)
     else:
         with open(filename, 'w') as file:
             json.dump(data, file)
+
+helpstring = """
+    Command-line utility to generate an input for chemiscope  - an interactive 
+    structure-property explorer. Parses an input file containing atomic structures
+    using the ASE I/O module, and converts it into a JSON file that can be loaded in
+    chemiscope. Frame and environment properties must be written in the same file 
+    containing atomic structures: we recommend the extended xyz format, which is 
+    flexible and simple. In all cases, this utility will simply transfer to the
+    JSON input anything that is readable by ASE. 
+"""
+            
+if __name__ == '__main__':
+
+    # command-line execution. requires ASE IO module
+    import argparse
+    try:
+        import ase.io as ase_io
+    except:
+        raise ImportError("chemiscope_input needs ASE modules to parse structure inputs")
+    
+    parser = argparse.ArgumentParser(description=helpstring)
+    parser.add_argument('-i', '--input', type=str, required=True,
+                        help='Input file name.')
+    parser.add_argument('-o', '--output', type=str, 
+                        help='Chemiscope JSON output file.')
+    parser.add_argument('-c', '--cutoff', type=float, 
+                        help='Generate  atom-centred environments with the \
+                              given cutoff.')                        
+    parser.add_argument('--name', default="", type=str,
+                        help='Dataset name.')
+    parser.add_argument('--description', default="", type=str,
+                        help='Dataset description.')
+    parser.add_argument('--authors', nargs='*', type=str, default=[],
+                        help='List of dataset authors.')
+    parser.add_argument('--references', nargs='*', type=str, default=[],
+                        help='List of references for the dataset.')                 
+    args = parser.parse_args()
+
+    # reads filen with ASE    
+    frames = ase_io.read(args.input, ':')
+    
+    # determines output file name automatically if missing
+    output = args.output
+    if args.output is None:
+        output = args.input + "_chemiscope.json"
```

Comment:
As a default, I would rather use `.json.gz` as the default extension, since there are no drawback and size advantages

Reply:
Also, shouldn't we remove the input extension? Otherwise, it can become (minorly) cumbersome with "file.xyz.json.gz" when one might prefer "file.json.gz"

Reply:
If someone prefer another extension, they can set it explicitly with `--output`. I would not try to deal with file extension here, it can be hard to do correctly (for example, what should we do with `file.xyz.gz`, or `HISTORY` DL_POLY files?)

Reply:
And I realize my initial comment was not very clear, I meant appending `_chemiscope.json.gz` to create the output file name, instead of `_chemiscope.json`

Reply:
Here I've mixed feelings. I avoided stripping the extension just because I didn't know how to do that. Perhaps ASE provides a way to remove the extension based on the same logic it uses to detect file format? otherwise we can just drop whatever comes after the last dot.  "DOS style"

Reply:
Since this is just a default and the user can always specify an output file name, I would keep it like this and don't try to deal with file extension.

Reply:
Fine with me